Foundation/URLSession/URLSessionTask.swift

```diff
@@ -376,7 +376,7 @@ open class URLSessionTask : NSObject, NSCopying {
     /*
      * The current state of the task within the session.
      */
-    open var state: URLSessionTask.State {
+    open fileprivate(set) var state: URLSessionTask.State {
```

Comment:
[`URLSessionTask.state` is get-only in Darwin Foundation](https://developer.apple.com/documentation/foundation/urlsessiontask/1409888-state#).